### PR TITLE
fix `VertxFileStorage` verification tests for case root sub storage

### PR DESCRIPTION
**Pull request by dgarus**
part of #445

Fixed VertxFileStorage verification tests for case root sub storage

### CODE REVIEW

File: asto-core/src/main/java/com/artipie/asto/Key.java [hunk 1]
```diff
@@ -53,11 +53,39 @@ public interface Key {
      */
     List<String> parts();
 
+    /**
+     * Base key class.
+     * @since 1.14.0
+     */
+    abstract class BaseKey implements Key {
```
Comment (olenagerasimova) [inline asto-core/src/main/java/com/artipie/asto/Key.java:60]: I'd suggest to call this class just `Base` without `key` as this class is a part of `Key` interface and will always be accessed as `Key.Base`.

Comment (dgarus) [reply asto-core/src/main/java/com/artipie/asto/Key.java]: ok

**Review by olenagerasimova — APPROVED** [review]
@dgarus thanks

**Comment by dgarus** [comment]
@olenagerasimova thank you for the review!

**Pull request merged**